comms/lrzsz/Portfile

```diff
@@ -26,6 +26,14 @@ configure.args      --program-transform-name=s/l// \
                     --mandir=${prefix}/share/man \
                     --disable-nls
 
+# workaround for clang 16
+if {[string match *clang* ${configure.cc}]} {
+    set clang_version [exec ${configure.cc} --version | grep -o -E {(clang|LLVM) version [0-9]+} | awk {{print $3}}]
+    if {${clang_version} >= 16} {
+        configure.cflags-append -Wno-implicit-int
+    }
```

Comment:
I'm not sure you need to check the Clang version. Instead, you can add the flag unconditionally [for Clang builds], along with  flags to ensure the option is ignored on earlier versions.

This is the pattern I use in some of my ports:

```
if { [string match *clang* ${configure.compiler}] } {
    configure.cflags-append \
                    -Wno-implicit-int \
                    -Wno-error=unknown-warning-option \
                    -Wno-unknown-warning-option
}
```

Though checking the Clang version is fine too. (With the caveat that you'll want to check `configure.compiler`, rather than `configure.cc`, for your initial Clang check.)

Reply:
Your advice to use compiler flags, a variation of my original change, conflicts with the advise of @reneeotten who suggests patching the code.  I'm curious if you have any feedback or additional input on the subject considering the conflicting advise. Thanks.

Reply:
> Your advice to use compiler flags, a variation of my original change, conflicts with the advise of @reneeotten who suggests patching the code.  I'm curious if you have any feedback or additional input on the subject considering the conflicting advise. Thanks.

Ultimately I defer to Renee, who is correct. Patching the configure script is the way to go.

Reply:
I shall do that then. Thanks for the quick reply.